JhipsterXamarin/JhipsterXamarin/JhipsterXamarin/Services/IAuthenticationService.cs

```diff
@@ -9,5 +10,6 @@ public interface IAuthenticationService
         UserModel CurrentUser { get; set; }
         Task<bool> SignIn(LoginModel model);
         void SignOut();
+        Task SetUserAndAuthorizationHeader(JwtToken token, bool save = false);
```

Comment:
are you sure of the visibility ? Who need it ?

Reply:
I forgot to remove it, removed in 77aae89 & updated to public a752d6a